server_test.go

```diff
@@ -9,11 +9,23 @@ import (
 	"net"
 	"reflect"
 	"testing"
+	"time"
 
 	"github.com/decred/dcrd/addrmgr/v2"
 	"github.com/decred/dcrd/wire"
 )
 
+var zeroTime = time.Time{}
+
+var (
```

Comment:
These should be defined inside the test scope (unless they're going to be used in a future commit).